### PR TITLE
chore(release): mark v7.0.0

### PR DESCRIPTION
**Pull request by jschfflr**

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]
LGTM but let's wait until Maksim's change landed

File: CHANGELOG.md
Comment (sadym-chromium) [inline CHANGELOG.md:11]:
```suggestion
* **chromium:** - `page.screenshot` makes a screenshot with the clip dimensions, not cutting it by the ViewPort size.
```

**Comment by sadym-chromium** [comment]
#6805 is merged. I added the suggestion to update the `CHANGELOG.md`. Despite that you are good to go. 

LGTM

**Review by sadym-chromium — APPROVED** [review]

**Comment by sadym-chromium** [comment]
The failing test is removed in the #6805 . Rebase will fix the CI.

File: CHANGELOG.md
Comment (mathiasbynens) [reply CHANGELOG.md:11]: The changelog is automatically populated based on the commit messages. If we make changes now they will be overwritten later.

**Comment by jschfflr** [comment]
@sadym-chromium Does the cl look good to you now?

**Comment by limonte** [comment]
Two major releases in two days, interesting approach.

**Comment by sadym-chromium** [comment]
We released Puppeteer v6.0.0 with Chromium 89 and v7.0.0 with Chromium 90. Each of them had breaking changes, that's why major version bump needed in both cases.

**Comment by limonte** [comment]
> We released Puppeteer v6.0.0 with Chromium 89 and v7.0.0 with Chromium 90. Each of them had breaking changes, that's why major version bump needed in both cases.

Got it, thanks for the clarification!

**Pull request merged**
